airflow/providers/mongo/hooks/mongo.py

```diff
@@ -210,7 +210,7 @@ def insert_one(
         return collection.insert_one(doc, **kwargs)
 
     def insert_many(
-        self, mongo_collection: str, docs: dict, mongo_db: str | None = None, **kwargs
+        self, mongo_collection: str, docs: list[dict], mongo_db: str | None = None, **kwargs
```

Comment:
Should this be Iterable instead? That's what I see in mongo at least.

Reply:
Thanks, you are right. I changed it to `Iterable`